nsca_client/models/nsca_server.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# © 2015 ABF OSIELL <http://osiell.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields
+
+
+class NscaServer(models.Model):
```

Comment:
isnot there any module that declare servers and that we could use here and in other modules?
It is a nice to have but I would rather create a server_setup module for that that we could reuse and expand (odoo instance, DNS server etc.)